### DISCUSSION TITLE
Can't install Manager

### DISCUSSION THREAD

**Discussion by pappajohndd** [Security-Onion-Solutions/securityonion#2309] [Unsupported Versions] [answered]
tried using CentOS iso, was getting an error about authentication failure. Next tried installing Ubuntu 18.04 and cloned from Git. Now getting "there is a problem downloading the so-acng:2.3.100 image"

**Comment by dougburks** [comment, marked as answer]
When you say `CentOS iso`, are you referring to our Security Onion ISO image based on CentOS or are you referring to a standard CentOS ISO?

What was the exact authentication failure error?

Have you double-checked that you are following all instructions in the documentation?
https://docs.securityonion.net/en/2.3/installation.html

**Reply by pappajohndd** [reply]
Yes using security onion iso. Initially got the manager to install without issue but could not get a search node to install, Getting "Attempt to authenticate with the salt master failed with timeout error". Decided to start over with manager thinking it was an issue with the manager. Then the failure started happening when installing the manager.

**Reply by pappajohndd** [reply]
Rebooted my hyper-v and got the manager installed with no issues. Installing search node now. Looks to be issue on my end.
